### ISSUE TITLE
Image Labels 

### ISSUE THREAD

**Issue by HostGuest** [rohanpillai20/Image-Classification-by-Keras-and-Tensorflow#9]
why keras code doesn't have labels for images ?
how the model this is chair or somthing else without labeling , i mean in traning set ?

**Comment by rohanpillai20** [comment]
I don't understand. You need to arrange the training folder in such a way that the script automatically understands the labels. For example, if you want to train a set of images of chairs, make a folder names "Chairs" and put images of chairs inside that folder. And so on and so forth for all other classes. Let me know if you have any more questions.
